equinox-shipping/Domain/Container.fs

```diff
@@ -0,0 +1,57 @@
+module Container
+
+// Container state needs to be serializable as it will be stored as part of the
+// Snapshotted event data.
+type ContainerState = { created: bool; finalized: bool }
+
+module Events =
+
+    let [<Literal>] CategoryId = "Shipment"
+    let streamName clientId = FsCodec.StreamName.create CategoryId clientId
```

Comment:
probably not a clientId ?
suggest defining a UMX type for whatever it is

Reply:
Made used of UMX thanks!